src/Common/src/System/Net/Http/TlsCertificateExtensions.cs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Net.Http
+{
+    /// <summary>
+    /// Defines Extension methods which are meant to be re-used across WinHttp & UnixHttp Handlers
```

Comment:
GitHub is nicely pointing out that the doc comments are HTML, and this free-floating ampersand is invalid HTML.  Please either change it to "and" or "&amp;amp;"